test/assert.js

```diff
@@ -120,6 +120,83 @@ test('.not()', function (t) {
 });
 
 test('.deepEqual()', function (t) {
```

Comment:
Maybe add a comment that these tests are here to detect regressions when the underlying comparison library changes?